R/export_cdisc.R

```diff
@@ -93,19 +100,38 @@ export_cdisc <- function(res_nca) {
     #  Recode PPTESTCD PKNCA names to CDISC abbreviations
     mutate(
       PPTESTCD = translate_terms(PPTESTCD, mapping_col = "PKNCA", target_col = "PPTESTCD"),
+      PPTEST = translate_terms(PPTESTCD, mapping_col = "PPTESTCD", target_col = "PPTEST"),
       DOMAIN = "PP",
       # Group ID
-      PPGRPID =  paste(ANALYTE, PCSPEC, paste("CYCLE", DOSNO,  sep = " "), sep = "-"),
+      PPGRPID = {
+        if ("PCGRPID" %in% names(.)) PCGRPID
+        else if ("AVISIT" %in% names(.)) paste(ANALYTE, DRUG, PCSPEC, AVISIT, sep = "-")
+        else paste(DRUG, PCSPEC, DOSNO, sep = "-")
+      },
```

Comment:
```suggestion
      PPGRPID =  dplyr::case_when(
        "AVISIT" %in% names(.) ~ paste0(ANALYTE, PCSPEC, AVISIT, sep = "-")
        "VISIT" %in% names(.) ~ paste0(ANALYTE, PCSPEC, VISIT, sep = "-")
        .default ~ paste(ANALYTE, PCSPEC, DOSNO, sep = "-")
      ),
```
Specs:
For all parameters, create a value comprised of the parameter category [PREPP.PPASTYP], parameter specimen [PREPP.PPASMED] and Visit [PREPP.PPVISIT] (separated by a '-').
For example. RRO4893594-SERUM-DAY 1

no need of DRUG as the NCA is done per metabolite
I guess VISIT could be there in some cases when AVISIT is not available

Reply:
@Gero1999 
1. What is important to know in PP or ADPP is one which Analyte the NCA has been run, so we don't need the Drug information, the matching DRUG + ANALYTE is only needed for the derivation of times in the data preparation, then you don't need it at all in the PK parameters. As the specs mention, we need PPASTYP (analyte), PPASMED (ppscpec) and PPVISIT (avisit), that is enough.
2. Do you mean that the ifelse would not crash while the case_when would?
3. Fine

Reply:
1. Ok then I will change
2. Yep 
3. On it! 💪🏽